io/io/inc/TMemFile.h

```diff
@@ -33,7 +36,8 @@ class TMemFile : public TFile {
       UChar_t   *fBuffer;
       Long64_t   fSize;
    };
-   TMemBlock    fBlockList;   ///< Colletion of memory blocks of size fBlockSize
+   TMemBlock    fBlockList;   ///< Collection of memory blocks of size fgDefaultBlockSize
+   const std::shared_ptr<const std::vector<char>> fExternalData; // shared file data / content
```

Comment:
The comment 'prefix' should be ///< to match the others.

Reply:
Please use an alias to reduce the typename length so that the members can be aligned.